src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -14,6 +16,7 @@
 public class PersonCard extends UiPart<Region> {
 
     private static final String FXML = "PersonListCard.fxml";
+    private static String[] DEFAULT_COLORED_TAGS = new String[]{"friends", "colleagues", "family", "neighbours"};
```

Comment:
Might I suggest using [Arrays.asList(T... a)](https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#asList-T...-) instead? eg. `List<String> DEFAULT_COLOURED_TAGS = Arrays.asList("friends", "colleagues"...)`. 

This way, you can directly use `DEFAULT_COLOURED_TAGS.contains(tag)` in line 76, which is less verbose.